File: docs/components_page/components/input/size.jl
using DashBootstrapComponents

inputs = html_div([
    dbc_input(placeholder = "A large input...", size = "lg", className = "mb-3"),
    dbc_input(placeholder = "A medium input...", size = "md", className = "mb-3"),
    dbc_input(placeholder = "A small input...", size = "sm"),
]);

File: docs/components_page/components/input/size.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

inputs <- htmlDiv(
  list(
    dbcInput(
      placeholder = "A large input...", size = "lg", className = "mb-3"
    ),
    dbcInput(
      placeholder = "A medium input...", size = "md", className = "mb-3"
    ),
    dbcInput(placeholder = "A small input...", size = "sm")
  )
)
